### PR TITLE
Add JSON LD descriptions for image/video blocks

### PR DESCRIPTION
**Pull request by nathggns**
## 🌟 What is the purpose of this PR?

This PR adds JSON LD descriptions for the image/video block.

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1203801045329780/1204217934265155/f) _(internal)_

## 🚫 Blocked by

N/A

## 🔍 What does this change?

- Add a JSON LD script tag for image block setting url/contentUrl to the src of the image, and description to the caption
- Add a JSON LD script tag for video block setting url/contentUrl to the src of the video, and description to the caption

## 📜 Does this require a change to the docs?

N/A

## ⚠️ Known issues

**Please see limitations outlined in Asana**

## 🐾 Next steps

**Please see limitations outlined in Asana**

## 🛡 What tests cover this?

None

## ❓ How to test this?

Check the HTML output of the blocks. 

We'll check how the output is recognised by bots later.

### CODE REVIEW

**Comment by nathggns** [comment]
@CiaranMn as described on Asana, could you take a look at whether this is sufficient to fulfil the task and I'll open the PR for proper review if so.

File: blocks/image/src/components/media.tsx [hunk 1]
```diff
@@ -262,37 +262,57 @@ export const Media: FunctionComponent<
...
+          /* eslint-disable-next-line react/no-danger */
+          dangerouslySetInnerHTML={{
+            __html: `
+            {
+              "@context": "https://schema.org/",
+              "@type": "ImageObject",
+              "@id": "${draftSrc}",
+              "url": "${draftSrc}",
+              "contentUrl": "${draftSrc}",
+              "description": "${draftCaption}"
```
Comment (CiaranMn) [inline blocks/image/src/components/media.tsx:310]: I think we should look at sanitizing this in case the source / caption is malicious

**Comment by nathggns** [comment]
@CiaranMn has verified IRL that this fulfils the requirements of the ticket

File: blocks/image/src/components/media.tsx (hunk 1 above)
Comment (nathggns) [reply blocks/image/src/components/media.tsx]: Great catch 🤦

File: blocks/video/src/components/media.tsx [hunk 2]
```diff
@@ -242,35 +242,54 @@ export const Media: FunctionComponent<
...
+      </div>
+      {draftSrc ? (
+        <script
+          type="application/ld+json"
+          /* eslint-disable-next-line react/no-danger */
+          dangerouslySetInnerHTML={{
+            // @note – using JSON.stringify to prevent potential XSS
+            __html: JSON.stringify({
+              "@context": "https://schema.org/",
+              "@type": "VideoObject",
```
Comment (CiaranMn) [inline blocks/video/src/components/media.tsx:284]: sorry to drip-feed comments in, but can we add a note / `@todo` calling out the fact that this is missing [required properties](https://developers.google.com/search/docs/appearance/structured-data/image-license-metadata#structured-data-type-definitions)?

Comment (nathggns) [reply blocks/video/src/components/media.tsx:285]: Done

**Review by CiaranMn — APPROVED** [review]

**Pull request merged**